torch_geometric/sampler/base.py

```diff
@@ -90,3 +92,12 @@ def sample_from_edges(
         r"""Performs sampling from the edges specified in 'index', returning
         a sampled subgraph in the specified output format."""
         raise NotImplementedError
+
+    @property
+    def edge_permutation(self) -> Union[OptTensor, Dict[EdgeType, OptTensor]]:
+        r"""If the sampler performs any modification of edge ordering in the
```

Comment:
Should we document the behaviour for HeteroGraphs too? Might not be clear why it can by a Dictionary.